### PR TITLE
Floof branch/master

### PR DESCRIPTION
**Pull request by TuckTuckFloof**
## Description

I added all of the files from EU ISO, I was wondering why it didn't work and it turns out I was trying to push master lmao. ANYWAY, I can see literally all of my commits from my creative process of EU ISO so I guess I'm here to make sure this all gets pushed onto the main branch! If you don't know what it is, it's basically a planck with some isos slapped on the side, greatest thing I've ever made!

## Types of Changes

- [x] New feature
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

### CODE REVIEW

**Comment by ridingqwerty** [comment]
There are some issues with this submission.  I don't have a problem with the personal keymap changes being included with a new board PR (although it's easier and cleaner if such changes are done as separate PRs), but there are changes to some unrelated default board files and some other folks' keymaps such that this probably can't be accepted as-is.

**Comment by tzarc** [comment]
At this stage, you're going to need to:

1. Remove all changes unrelated to `keyboards/EU_ISOlation`
2. Rename it to `keyboards/eu_isolation` as per QMK naming standards.

You've also removed the entire set of the PR checkboxes, so at this point I'm inclined to just close this PR and ask you to create a new PR once you've made the modifications as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist). If you feel like recovering the content of the PR description, feel free, but this is likely going nowhere until that's done.

We have a process that needs to be followed.

**Comment by TuckTuckFloof** [comment]
Hello, yea, I can do that, I already changed the name of the folder. I'd
say nuke it if that entails what I would assume would be yeeting all the
commits. Thanks for the feedback, this is my first PR here.

File: keyboards/EU_ISOlation/config.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```
Comment (drashna) [inline keyboards/EU_ISOlation/config.h:8]:
```suggestion
#define PRODUCT_ID      0x4373
```

From hash generated:
```sh
echo -n "EU ISOlation" | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
```

File: keyboards/EU_ISOlation/config.h [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    TuckTuckFloof
+#define PRODUCT         EU ISOlation
+#define DESCRIPTION     A terrible fucking creation involving a quad iso macro pad
```
Comment (drashna) [inline keyboards/EU_ISOlation/config.h:12]: This is not used, and should be removed. 
```suggestion
```

File: keyboards/EU_ISOlation/info.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
+{
+    "keyboard_name": "EUISOlation",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 14,
+    "height": 4,
+    "layouts": {
+        "LAYOUT_EUISO_ORTHO": {
```
Comment (drashna) [inline keyboards/EU_ISOlation/info.json:8]:
```suggestion
        "LAYOUT_euiso_ortho": {
```

File: keyboards/EU_ISOlation/info.json [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+                {"x":6, "y":3},
+                {"x":7, "y":3},
+                {"x":8, "y":3},
+                {"x":9, "y":3},
+                {"x":10, "y":3},
+                {"x":11, "y":3}
+            ]
+        },
+
+        "LAYOUT_EUISO_MIT": {
```
Comment (drashna) [inline keyboards/EU_ISOlation/info.json:66]:
```suggestion
        "LAYOUT_euiso_mit": {
```

File: keyboards/EU_ISOlation/info.json [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+                {"x":5, "y":3, "w":2},
+                {"x":7, "y":3},
+                {"x":8, "y":3},
+                {"x":9, "y":3},
+                {"x":10, "y":3},
+                {"x":11, "y":3}
+            ]
+        },
+
+        "LAYOUT_EUISO_BIGSPACE": {
```
Comment (drashna) [inline keyboards/EU_ISOlation/info.json:123]:
```suggestion
        "LAYOUT_euiso_bigspace": {
```

File: keyboards/EU_ISOlation/keymaps/FloofBS/config.h [hunk 6]
```diff
@@ -0,0 +1,2 @@
+#define COMBO_COUNT 5
```
Comment (drashna) [inline keyboards/EU_ISOlation/keymaps/FloofBS/config.h:1]:
```suggestion
#pragma once

#define COMBO_COUNT 5
```

And could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/EU_ISOlation/keymaps/FloofBS/keymap.c [hunk 7]
```diff
@@ -0,0 +1,80 @@
+#include "EU_ISO.h"
```
Comment (drashna) [inline keyboards/EU_ISOlation/keymaps/FloofBS/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15


Also,
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/EU_ISOlation/keymaps/default_BIGSPACE/config.h [hunk 8]
```diff
@@ -0,0 +1,2 @@
+#define COMBO_COUNT 5
```
Comment (drashna) [inline keyboards/EU_ISOlation/keymaps/default_BIGSPACE/config.h:1]:
```suggestion
#pragma once 

#define COMBO_COUNT 5
```

File: keyboards/EU_ISOlation/keymaps/default_BIGSPACE/keymap.c [hunk 9]
```diff
@@ -0,0 +1,45 @@
+#include "EU_ISO.h"
```
Comment (drashna) [inline keyboards/EU_ISOlation/keymaps/default_BIGSPACE/keymap.c:1]:
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/EU_ISOlation/keymaps/default_ORTHO/keymap.c [hunk 10]
```diff
@@ -0,0 +1,45 @@
+#include "EU_ISO.h"
```
Comment (drashna) [inline keyboards/EU_ISOlation/keymaps/default_ORTHO/keymap.c:1]:
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/eu_isolation/EU_ISO.h [hunk 11]
```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "quantum.h"
+
+#define ___ KC_NO
+
+#define LAYOUT_EUISO_MIT( \
```
Comment (drashna) [inline keyboards/eu_isolation/EU_ISO.h:7]:
```suggestion
#define LAYOUT_euiso_mit( \
```

File: keyboards/eu_isolation/EU_ISO.h [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+    k32, k33, k34, k35, k36,      k38, k39, k3a, k3b, k3c, k3d \
+) \
+{ \
+    { ___, ___, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d }, \
+    { ___, ___, k32, k33, k34, k35, ___, k37, k38, k39, k3a, k3b, k3c, k3d } \
+}
+
+#define LAYOUT_EUISO_ORTHO( \
```
Comment (drashna) [inline keyboards/eu_isolation/EU_ISO.h:20]:
```suggestion
#define LAYOUT_euiso_ortho( \
```

File: keyboards/eu_isolation/EU_ISO.h [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+    k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d  \
+) \
+{ \
+    { ___, ___, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d }, \
+    { ___, ___, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d } \
+}
+
+#define LAYOUT_EUISO_BIGSPACE( \
```
Comment (drashna) [inline keyboards/eu_isolation/EU_ISO.h:33]:
```suggestion
#define LAYOUT_euiso_bigspace( \
```

File: keyboards/eu_isolation/config.h [hunk 14]
```diff
@@ -0,0 +1,33 @@
+
```
Comment (drashna) [inline keyboards/eu_isolation/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/40percentclub/gherkin/config.h [hunk 15]
```diff
@@ -15,15 +15,15 @@
...
+#define MATRIX_ROW_PINS { C6, D7, E6, B4, B5 }
+#define MATRIX_COL_PINS { B2, B3, B1, F7, F6, F5 }
 #define UNUSED_PINS
 
 /* COL2ROW or ROW2COL */
 #define DIODE_DIRECTION COL2ROW
 
 /* number of backlight levels */
-#define BACKLIGHT_PIN B5
+#define BACKLIGHT_PIN B6
```
[the remark this answers is not in the document]
Comment (TuckTuckFloof) [reply keyboards/40percentclub/gherkin/config.h:26]: That was an attempt to fix a usb-c I put on backwards lol. It didn't work...

**Comment by drashna** [comment]
the branch is still intact. but you'd need to checkout `floofBranch/master` instead of just `master`

**Pull request closed without merging**